### PR TITLE
Add appcompat-v7 ActionMenuView bindings

### PR DESCRIPTION
**Pull request by dlackty**

### CODE REVIEW

File: rxbinding-appcompat-v7/src/main/java/com/jakewharton/rxbinding/support/v7/widget/RxActionMenuView.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+     */
+    @CheckResult @NonNull
+    public static Observable<MenuItem> itemClicks(@NonNull ActionMenuView view) {
+        checkNotNull(view, "view == null");
+        return Observable.create(new ActionMenuViewItemClickOnSubscribe(view));
+    }
+
+    private RxActionMenuView() {
+        throw new AssertionError("No instances.");
+    }
```
Comment (JakeWharton) [inline rxbinding-appcompat-v7/src/main/java/com/jakewharton/rxbinding/support/v7/widget/RxActionMenuView.java]: This file should be indented with two spaces, not four.

File: rxbinding-appcompat-v7/src/main/java/com/jakewharton/rxbinding/support/v7/widget/ActionMenuViewItemClickOnSubscribe.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        };
+        view.setOnMenuItemClickListener(listener);
+
+        subscriber.add(new MainThreadSubscription() {
+            @Override
+            protected void onUnsubscribe() {
+                view.setOnMenuItemClickListener(null);
+            }
+        });
+    }
```
Comment (JakeWharton) [inline rxbinding-appcompat-v7/src/main/java/com/jakewharton/rxbinding/support/v7/widget/ActionMenuViewItemClickOnSubscribe.java]: This file should be indented with two spaces, not four.

File: rxbinding-appcompat-v7/src/main/java/com/jakewharton/rxbinding/support/v7/widget/ActionMenuViewItemClickOnSubscribe.java [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+import android.support.v7.widget.ActionMenuView;
+import android.view.MenuItem;
+
+import rx.Observable;
+import rx.Subscriber;
+import rx.android.MainThreadSubscription;
+
+import static com.jakewharton.rxbinding.internal.Preconditions.checkUiThread;
+
+public final class ActionMenuViewItemClickOnSubscribe implements Observable.OnSubscribe<MenuItem> {
```
Comment (JakeWharton) [inline rxbinding-appcompat-v7/src/main/java/com/jakewharton/rxbinding/support/v7/widget/ActionMenuViewItemClickOnSubscribe.java]: -`public`

**Comment by JakeWharton** [comment]
Just minor comments. Looks great otherwise!

**Comment by dlackty** [comment]
@JakeWharton Updated & squashed. Thanks for reviewing!

**Comment by JakeWharton** [comment]
Looks like just you need to run `./gradlew gK` to regenerate the Kotlin bindings.

**Comment by dlackty** [comment]
@JakeWharton Done. :ok_hand:

**Comment by JakeWharton** [comment]
Thanks!

**Pull request merged**